node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -126,17 +126,14 @@ impl<C: Context> ActiveValidator<C> {
     pub(crate) fn on_new_vote(
         &mut self,
         vhash: &C::Hash,
-        mut timestamp: Timestamp,
+        timestamp: Timestamp,
```

Comment:
Should we call this `now`, too? (Also in `should_send_confirmation`.)

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/354/commits/cdd18db9cea6cc1df8ea8cff9b2245fbd4637898